pkg/renderer/kubernetes_helm_test.go

```diff
@@ -169,6 +169,16 @@ func TestRenderSensorHelm(t *testing.T) {
 	}
 }
 
+func TestRenderSensorTLSSensorOnly_NotPanicOnMissingImageData(t *testing.T) {
+	fields := getDefaultMetaValues(t)
+	fields.CertsOnly = true
+	// (ROX-16212) Should not fail when meta-values don't set ImageTag (e.g. when running with Operator installation)
+	// ImageTag isn't used to render TLS secrets, therefore it shouldn't result in RenderSensorTLSSecretsOnly returning an error
+	fields.ImageTag = ""
+	_, err := RenderSensorTLSSecretsOnly(*fields, certs)
```

Comment:
How about asserting that the resulting yaml has the `should-never-use` value?

Reply:
(*) "does not have that substring"

+1 to the suggestion.

Reply:
Done, check latest commit